docs/ref/models/fields.txt

```diff
@@ -1343,12 +1343,12 @@ following optional arguments:
 .. attribute:: ImageField.height_field
 
     Name of a model field which will be auto-populated with the height of the
-    image each time the model instance is saved.
+    image each time the image attribute is set.
```

Comment:
Wondering if "image attribute" is the best way to say this. What do you think about...
```suggestion
    image each time the image is set.
```
or 
```suggestion
    image each time the :class:`ImageField` instance is set.
```

Reply:
I'd like to avoid directly referencing the term `ImageField`, because it's _annoyingly_ complicated exactly what's going on.

I chose to use the word "attribute" because the code specifically calls ImageFieldFile `attr_class`.

If you omit the word attribute, you can still "set the image" and not trigger the descriptor or the width/height logic.

Here's an example of "setting the image" where you're not "setting the image attribute":

```python
MyModel.objects.filter(id=3).update(photo="myphoto.jpg")
```

So I'm `-1` on the second change and `-0` on the first one.

Reply:
I should have clarified why I am not keen on "image attribute" here. 
In the context of an attribute definition (`height_field`) referring to an "image attribute" makes it reasonable for readers to believe there is an undocumented attribute called `image`.

`attr_class` is referenced in the docs in a section "Writing a ``FileField`` subclass", but in a different page, so I think reader knowledge of `attr_class` is unlikely in order to make this connection. I understand why you chose to name it as such.

`FieldFile` is documented but not `ImageFieldFile` which might have been nice to reference (as there is a section refering to `FieldFile` vs `FileField`). 

I feel like that `.update` bypasses logic is fairly well know but that's a valid point. 
Let me try again, how about...
```
    Name of a model field which is auto-populated with the height of the image 
    each time an image object is set.
```
If you are 0+ I can push up the tweaks and merge 👍

Reply:
I think that's "good enough" / `+0`. Go ahead and push/merge